reference/tools/cmake/cmaketoolchain.rst

```diff
@@ -607,6 +607,16 @@ CMakeToolchain is affected by these ``[conf]`` variables:
 - **tools.cmake.cmaketoolchain:system_name** is not necessary in most cases and is only used to force-define ``CMAKE_SYSTEM_NAME``.
 - **tools.cmake.cmaketoolchain:system_version** is not necessary in most cases and is only used to force-define ``CMAKE_SYSTEM_VERSION``.
 - **tools.cmake.cmaketoolchain:system_processor** is not necessary in most cases and is only used to force-define ``CMAKE_SYSTEM_PROCESSOR``.
+- **tools.cmake.cmaketoolchain:extra_variables**: dict-like python object which specifies the CMake variable name and value. E.g.
+    
+.. code:: bash
+
+    conan install . -c "tools.cmake.cmaketoolchain:extra_variables*={'CMAKE_GENERATOR_INSTANCE': 'C:/Program Files (x86)/Microsoft Visual Studio/2017/BuildTools/'}"
+
+.. tip::
```

Comment:
TIL! Can I get a pic of how this looks when built?

Reply:
![image](https://github.com/conan-io/docs/assets/26503843/9740ad65-2ecd-4f4b-9836-e517f484a0cd)
It looks good to me, but maybe it is not what we want.
As a noobie, I like command examples so that's why I added it. And also the *= looks very important because really nobody read the docs xD

Reply:
Update: 
![image](https://github.com/conan-io/docs/assets/26503843/d2e91a65-1ef8-4e0e-8118-0c3cc6b65d56)

Reply:
Final result
![image](https://github.com/conan-io/docs/assets/26503843/48e78416-4e7f-45ca-bcdf-36c98791193d)